meetings/2018-11-12.md

```diff
@@ -0,0 +1,65 @@
+# Node.js Foundation Bootstrap Team Meeting 2018-11-12
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=dIRyJ2nzYCY
+* **GitHub Issue**: https://github.com/nodejs/bootstrap/issues/34
+
+## Present
+
+- Myles Borins (@MylesBorins)
+- Mike Dolan (@mkdolan)
+- Jory Burson (@jorydotcom)
+- Adam Miller (@amiller-gh)
+- Tracy Hinds (@hackygolucky)
+- Bruce MacNaughton (@bmacnaughton)
+- Abraham Jr. Agiri (@codeekage)
+- Michael Dawson (@mhdawson)
+- Brian Warner (@brianwarner)
+- Sakthipriyan Vairamani (@thefourtheye)
+
+## Agenda
+
+Extracted from **bootstrap-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### Approving PRs (3 minute timebox)
+
+* Draft Staging Process [#23](https://github.com/nodejs/bootstrap/pull/23)
+  - 3 minute timebox
+  - Jory introduced the process and noted that it can be simplified as necessary.
+  - Myles proposes moving forward once grammatical changes are landed
+  - Myles takes action to land
+
+### Stage Advancement (3 minute timebox)
+
+* Governance stage 1 [#32](https://github.com/nodejs/bootstrap/pull/32)
+  - 3 minute timebox
+  - Proposal to move governance from stage 0 to stage 1
+  - We have a general idea of how we’re doing it, but don’t need consensus on exact wording
+  - Myles takes action to land
+
+## Update on progress (4 minute timebox)
+
+* First Pass at Bootstrap Process [#7](https://github.com/nodejs/bootstrap/pull/7)
+  - 2 minute timebox
+  - Adam will close current PR.
+  - New PR to be opened by Adam on the README with the current contents migrated
+* Create expectations for foundation draft [#6](https://github.com/nodejs/bootstrap/pull/6)
+  - 2 minute timebox
+  - Open question: At what point do we consider the proposals firm enough to start getting feedback from lawyers and board members?
+  - Open question: How do we ensure that budget responsibility is sufficiently consolidated to ensure that all required actions are taken throughout the year? (establishing budget, ensuring project execution, etc)
+  - Tracy takes action to incorporate TSC and CommComm’s expectations of the board and the executive committee into PR
+
+### Discussion (45 minute timebox)
+
+* How often / when do we meet? [#21](https://github.com/nodejs/bootstrap/issues/21)
+  - 2 minute timebox
+  - Action to revisit in next meeting to find a time that works for Asia, Africa, and Europe, as well as EST and PST
+* Drafting initial governance for merged foundation [#33](https://github.com/nodejs/bootstrap/issues/33)
+  - 40 minute timebox
+  - Mike D takes action to make governance document public 
```

Comment:
There were two "Mike D"s on the call today 😉

Reply:
Good point, fixed now. :-)